src/SurveyService.Validation/Question/CreateGroupQuestionRequestValidator.cs

```diff
@@ -0,0 +1,28 @@
+using FluentValidation;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Question;
+
+public class CreateGroupQuestionRequestValidator : AbstractValidator<CreateGroupQuestionRequest>, ICreateGroupQuestionRequestValidator
```

Comment:
you use this validator just once in CreateGroupRequestValidator; what's the point to separate it?

Reply:
Just for readability